src/modules/auth/auth.controller.ts

```diff
@@ -76,9 +76,9 @@ export class AuthController {
     return this.authService.restorePassword(dto);
   }
 
-  @Post('avatar')
   @UseGuards(JwtAuthGuard)
-  @UseInterceptors(FileInterceptor('file'))
+  @Post('avatar')
+  @UseInterceptors(FileInterceptor('avatar'))
```

Comment:
So interceptor accepts all file types?

Reply:
Added the multer avatar options, now it only accepts jpeg, png, and jpg as it is in the frontend. Also added the size limitation of 5MB to it

```typescript
@UseInterceptors(FileInterceptor('avatar', multerAvatarOptions))


export const multerAvatarOptions: MulterOptions = {
  fileFilter(
    req: Request,
    file: Express.Multer.File,
    done: (error: Error, acceptFile: boolean) => void,
  ) {
    if (!file.mimetype.match(/\/(jpeg|jpg|png)$/)) {
      done(
        new HttpException('Invalid file type', HttpStatus.BAD_REQUEST),
        false,
      );
    }

    if (file.size > 5e6) {
      done(
        new BadRequestException(
          `The maximum file size has been exceeded. Please select a file smaller than 5MB`,
        ),
        null,
      );
    }

    done(null, true);
  },
};
```